### PR TITLE
Review Request

### PR DESCRIPTION
**Pull request by TechnicallyRy**
Review request for Lab 1 to merge

### CODE REVIEW

**Comment by apisdn** [comment]
gj :)

**Comment by apisdn** [comment]
gj :)

**Review by apisdn — APPROVED** [review]
gj :)

File: test/test_blink.c [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    bool led_is_on;
+    const struct device *dev = device_get_binding(LED);
+    int ret = led_setup(dev, &led_is_on, PIN, GPIO_OUTPUT_ACTIVE | FLAGS, &counter);
+    TEST_ASSERT_EQUAL_MESSAGE(0, ret, "LED was not set up properly.");
+    TEST_ASSERT_TRUE_MESSAGE(led_is_on, "LED should start toggled off");
+    toggle_LED(dev, PIN, &led_is_on, &counter);
+    TEST_ASSERT_TRUE_MESSAGE(!led_is_on, "LED should have changed to on.");
+    TEST_ASSERT_TRUE_MESSAGE(gpio_pin_get(dev, PIN), "GPIO should report on");
+}
+
```
Comment (jpgarey) [inline test/test_blink.c:31]: I feel like you'd need a test for led_setup, because it's your other function you created.

**Comment by jpgarey** [comment]
I put one comment in the code saying you could add a test for led_setup. I think it'd be good to have in full project so you can verify it doesn't break, but for this code it seems trivial. So I'm approving this. Good job guys, looks great.

**Review by jpgarey — APPROVED** [review]

**Comment by TechnicallyRy** [comment]
Verified review request.

**Pull request merged**